Scripts/dea_classificationtools.py

```diff
@@ -1113,52 +1123,58 @@ def SKCV(coordinates, n_splits, cluster_method, kfold_method,
         sklearn.mixture.GuassianMixture depending on the cluster_method argument.
-
     Returns
     --------
     generator object _BaseSpatialCrossValidator.split
-
-
     """
     # intiate a method
     if kfold_method == 'SpatialShuffleSplit':
-        splitter=_SpatialShuffleSplit(n_groups=n_groups,
-                                       method=cluster_method,
-                                       coordinates=coordinates,
-                                       max_distance=max_distance,
-                                       test_size=test_size,
-                                       train_size=train_size,
-                                       n_splits=n_splits,
-                                       random_state=random_state,
-                                       balance=balance,
-                                       **kwargs)
+        splitter = _SpatialShuffleSplit(n_groups=n_groups,
+                                        method=cluster_method,
+                                        coordinates=coordinates,
+                                        max_distance=max_distance,
+                                        test_size=test_size,
+                                        train_size=train_size,
+                                        n_splits=n_splits,
+                                        random_state=random_state,
+                                        balance=balance,
+                                        **kwargs)
 
     if kfold_method == 'SpatialKFold':
-        splitter=_SpatialKFold(n_groups=n_groups,
-                                coordinates=coordinates,
-                                max_distance=max_distance,
-                                method=cluster_method,
-                                n_splits=n_splits,
-                                random_state=random_state,
-                                balance=balance,
-                                **kwargs)
+        splitter = _SpatialKFold(n_groups=n_groups,
+                                 coordinates=coordinates,
+                                 max_distance=max_distance,
+                                 method=cluster_method,
+                                 test_size=test_size,
+                                 n_splits=n_splits,
+                                 random_state=random_state,
+                                 balance=balance,
+                                 **kwargs)
 
     return splitter
 
 
-def spatial_train_test_split(X, y, coordinates, cluster_method, kfold_method,
-                             test_size, balance, n_groups=None, max_distance=None,
-                             random_state=None, train_size=None, **kwargs):
+def spatial_train_test_split(X,
+                             y,
+                             coordinates,
+                             cluster_method,
+                             kfold_method,
+                             test_size,
```

Comment:
If you want to emphasise n_splits, consider making it non-optional, instead of making test_size non-optional.